WhatsNew.Cli/action.yml

```
@@ -0,0 +1,52 @@
+name: Clean repo
+description: Automatically synchronize GitHub issues with Quest (Azure DevOps).
+author: Bill Wagner
+# https://actions-cool.github.io/github-action-branding/
+branding:
+  icon: 'zap'
+  color: 'purple'
+inputs:
+  owner:
+    description: 'The GitHub organization name.'
+    required: true
+  repo:
+    description: 'The GitHub repository name within the provided organization.'
+    required: true
+  branch:
+    description: 'The branch name within the provided repository.'
+    default: main
+  docset:
+    description: 'The product name within the provided repository.'
+    default: 'false'
+  startdate:
+    description: 'A range start date in a valid format.'
+    required: true
+  enddate:
+    description: 'A range end date in a valid format.'
+    required: true
+  savedir:
+    description: 'An absolute directory path to which the generated Markdown file should be written.'
```

This can be relative:

```suggestion
    description: 'A directory path to which the generated Markdown file should be written.'
```